CMSgov/mint-app
# [EASI-3128] Discussion tagging FE

#EASI-3128

## Changes and Description

- Added TipTap RTE for mention capability
- `MentionTextArea` RTE component
  - Preloaded solution contacts while Okta async lookup fetches
  - Additional props for readonly, disabled, etc
- MentionList component (utilizing Tippy library) for dropdown
- Suggestion component for event handlers with async data/loading
- Updated TruncatedText component to integrate with readonly RTE



## How to test this change

Verify functionality matches [Figma](https://www.figma.com/file/tBnHaZl30hqsBqzsnRhul5/MINT---Discussions?type=design&node-id=1-22223&mode=design&t=MPJFd7RDUc6l2oxA-0)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
